packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -460,9 +473,6 @@ test('first', async t => {
     'unused collateral remains after liquidation',
   );
 
-  t.deepEqual(await E(vaultFactory).getPenaltyAllocation(), {
-    RUN: AmountMath.make(runBrand, 30n),
```

Comment:
I expected the new PenaltySeats to have accessors like this.  Why don't they?

Reply:
The plan is not to expose the penalty seat. @dtribble said not to use the fee collector pattern here and instead have the liquidation contract move the assets directly to the reserve. To give input on that: https://github.com/Agoric/agoric-sdk/issues/5367